src/theme.js

```diff
@@ -11,6 +11,11 @@ export const theme = {
         waterloo: "#7E839A",
     },
     breakpoints: {
-        mobile: 320,
+        mobile: "480px",
+        mobile2: "600px",
+        mobile3: "768px",
+        tablet: "1024px",
+        tablet2: "1200px",
+        laptop: "1480px"
```

Comment:
Good job! I would suggest to add one more breakpoint, for 320px width, for example: "mobileS: "320px"", just in case, and rename all breakpoints with number in name like tablet2 on something like that: tabletL, mobilieL, mobileXL, mobileM...